app/src/main/java/com/alphawallet/app/ui/TransactionDetailActivity.java

```diff
@@ -193,10 +194,14 @@ private void setupWalletDetails() {
         amount.setText(rawValue);
     }
 
-    private String getDate(long timeStampInSec) {
-        Calendar cal = Calendar.getInstance(Locale.ENGLISH);
-        cal.setTimeInMillis(timeStampInSec * 1000);
-        return DateFormat.getLongDateFormat(this).format(cal.getTime());
+    private String getDateAndTime(long timeStampInSec)
+    {
+        Calendar calendar = Calendar.getInstance(TimeZone.getDefault());
+        calendar.setTimeInMillis(timeStampInSec * DateUtils.SECOND_IN_MILLIS);
+        Date date = calendar.getTime();
```

Comment:
What's the difference between your code

        Calendar calendar = Calendar.getInstance(TimeZone.getDefault());
        calendar.setTimeInMillis(timeStampInSec * DateUtils.SECOND_IN_MILLIS);
        Date date = calendar.getTime();

 and the following textbook Java code?

        Date date = new java.util.Date(timeStampInSec*DateUtils.SECOND_IN_MILLIS);

Reply:
Second one looks better, and returns the same device locale time. I re-used the code from the existing code in LocaleUtils, as I didn't want to change anything we already set up. Since the second is better then it makes sense to also update the instance in LocaleUtils.